src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidateHandler.java

```diff
@@ -48,7 +48,8 @@ public boolean canHandle(
         return isHomeOfficeIntegrationEnabled
             && callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
             && (callback.getEvent() == Event.SUBMIT_APPEAL
-            || callback.getEvent() == Event.PAY_AND_SUBMIT_APPEAL)
+            || callback.getEvent() == Event.PAY_AND_SUBMIT_APPEAL
+            || callback.getEvent() == Event.MARK_APPEAL_PAID)
             && (callback.getCaseDetails().getState() == APPEAL_SUBMITTED
```

Comment:
Hi @divisathyan, Could you please take out these state checks as well to be in consistent with HO-integration-api?

Reply:
Removed state check and squashed all commits to one